src/server.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * This file is here only to show you how to proceed if you would
+ * like to turn your application into a standalone executable.
+ *
+ * You can launch it with the command `npm run standalone`
+ */
+
+import Fastify from "fastify";
+
+// Import library to exit fastify process, gracefully (if possible)
+import closeWithGrace from "close-with-grace";
+
+// Import your application as a normal plugin.
+import serviceApp from "./app.js";
+
+const environment = process.env.NODE_ENV ?? "production";
```

Comment:
Please don’t use NODE_ENV for anything. This is a massive antipattern.

https://youtu.be/HMM7GJC5E2o?si=d9idweist4369pBW

Reply:
Is this ok?
https://github.com/fastify/demo/pull/1/files#diff-8a8ae07582c9d433ec8c2e5c4310ff8901e604f4965c5b90a49117ad46c47595R21

```ts
function getLoggerOptions() {
  if (process.env.LOGGING === "pretty") {
    return {
      level: "info",
      transport: {
        target: "pino-pretty",
        options: {
          translateTime: "HH:MM:ss Z",
          ignore: "pid,hostname",
        },
      },
    };
  }

  return process.env.LOGGING === "default";
}
```